### DISCUSSION TITLE
Language model size.

### DISCUSSION THREAD

**Discussion by bert269** [toverainc/willow#248] [Q&A]
I noticed that with RC1 there has been a **LARGE_V2** language model released.
When flashing willow - do we need to code **large_v2** or will **large** still work to pick up v2?
This is my config setting: https://192.168.xxx.yyy:19000/api/willow?model=large&beam_size=5

**Comment by nikito** [comment]
As far as I am aware large is equal to large_v2, I don't think there is any
other large model. :)
Thanks and Best Regards,
Nick Bento

**Comment by kristiankielhofner** [comment]
WIS has always used large-v2. In configs, URI params, etc it is "large".

**Comment by bert269** [comment]
Thanks for classifying....
